src/main/java/core/basesyntax/figure/AbstractFigure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figure;
+
+public abstract class AbstractFigure implements Figure, Drawable {
```

Comment:
It is kind of stated in the declaration of the class that it is abstract :)
```suggestion
public abstract class Figure implements Figure, Drawable {
```

Reply:
I made changes.